[kiali] Module not found: Error: Can't resolve '../../assets/img/runtime/quarkus.svg'

### Describe the bug

After I followed the documentation to add the Kiali frontend plugin, running yarn dev resulted in the following error:

```shell
[0] ERROR in ../../node_modules/@janus-idp/backstage-plugin-kiali/dist/index.esm.js 6550:17-65
[0] Module not found: Error: Can't resolve '../../assets/img/mtls-status-full.svg' in '/Users/xx/code/backstage/octopus/node_modules/@janus-idp/backstage-plugin-kiali/dist'
[0] 
[0] ERROR in ../../node_modules/@janus-idp/backstage-plugin-kiali/dist/index.esm.js 6551:19-70
[0] Module not found: Error: Can't resolve '../../assets/img/mtls-status-partial.svg' in '/Users/xx/code/backstage/octopus/node_modules/@janus-idp/backstage-plugin-kiali/dist'
[0] 
[0] ERROR in ../../node_modules/@janus-idp/backstage-plugin-kiali/dist/index.esm.js 6552:21-74
[0] Module not found: Error: Can't resolve '../../assets/img/mtls-status-full-dark.svg' in '/Users/xx/code/backstage/octopus/node_modules/@janus-idp/backstage-plugin-kiali/dist'
[0] 
[0] ERROR in ../../node_modules/@janus-idp/backstage-plugin-kiali/dist/index.esm.js 6553:23-79
[0] Module not found: Error: Can't resolve '../../assets/img/mtls-status-partial-dark.svg' in '/Users/xx/code/backstage/octopus/node_modules/@janus-idp/backstage-plugin-kiali/dist'
[0] 
[0] ERROR in ../../node_modules/@janus-idp/backstage-plugin-kiali/dist/index.esm.js 10675:20-63
[0] Module not found: Error: Can't resolve '../../assets/img/api/graphql.svg' in '/Users/xx/code/backstage/octopus/node_modules/@janus-idp/backstage-plugin-kiali/dist'
```

I also did not find the corresponding SVG in the node_modules.

<img width="305" alt="image" src="https://github.com/janus-idp/backstage-plugins/assets/18688410/20cf5c55-5290-42be-a8ad-80017b2b78a3">


### Expected Behavior

### What are the steps to reproduce this bug?

1. Follow the [documentation](https://janus-idp.io/plugins/kiali/) to register the frontend and backend plugin
2. "Run yarn dev from the project's root directory

### Versions of software used and environment

```json
    "@janus-idp/backstage-plugin-kiali": "^1.16.4",
```

--- a/plugins/bulk-import-backend/package.json
+++ b/plugins/bulk-import-backend/package.json
@@ -58,7 +58,7 @@
     "@backstage/plugin-permission-common": "^0.8.0",
     "@backstage/plugin-permission-node": "^0.8.0",
     "@janus-idp/backstage-plugin-audit-log-node": "1.4.0",
-    "@janus-idp/backstage-plugin-bulk-import-common": "1.0.1",
+    "@janus-idp/backstage-plugin-bulk-import-common": "1.0.0",
     "@octokit/auth-app": "^6.0.3",
     "@octokit/core": "^5.1.0",
     "@octokit/rest": "^20.0.2",

--- a/plugins/kiali/dev/index.tsx
+++ b/plugins/kiali/dev/index.tsx
@@ -1,3 +1,5 @@
+/// <reference types="@backstage/cli/asset-types" />
+
 import React from 'react';
 
 import { Content, InfoCard, Page } from '@backstage/core-components';

--- a/plugins/kiali/src/components/Logos/Logos.tsx
+++ b/plugins/kiali/src/components/Logos/Logos.tsx
@@ -1,26 +1,22 @@
 import React from 'react';
 
+import GraphqlIcon from '../../assets/img/api/graphql.svg';
+import GrpcIcon from '../../assets/img/api/grpc.svg';
+import RestIcon from '../../assets/img/api/rest.svg';
+import GoLogo from '../../assets/img/runtime/go.svg';
+import JVMLogo from '../../assets/img/runtime/java.svg';
+import MicroProfileLogo from '../../assets/img/runtime/microprofile.svg';
+import NodejsLogo from '../../assets/img/runtime/nodejs.svg';
+import QuarkusLogo from '../../assets/img/runtime/quarkus.svg';
+import SpringBootLogo from '../../assets/img/runtime/spring-boot.svg';
+import ThorntailLogo from '../../assets/img/runtime/thorntail.svg';
+import TomcatLogo from '../../assets/img/runtime/tomcat.svg';
+import VertxLogo from '../../assets/img/runtime/vertx.svg';
 import { kialiStyle } from '../../styles/StyleUtils';
 
 const iconStyle = kialiStyle({
   height: '1.5rem',
 });
-
-const GraphqlIcon = require('../../assets/img/api/graphql.svg') as string;
-const GrpcIcon = require('../../assets/img/api/grpc.svg') as string;
-const RestIcon = require('../../assets/img/api/rest.svg') as string;
-const GoLogo = require('../../assets/img/runtime/go.svg') as string;
-const JVMLogo = require('../../assets/img/runtime/java.svg') as string;
-const MicroProfileLogo =
-  require('../../assets/img/runtime/microprofile.svg') as string;
-const NodejsLogo = require('../../assets/img/runtime/nodejs.svg') as string;
-const QuarkusLogo = require('../../assets/img/runtime/quarkus.svg') as string;
-const SpringBootLogo =
-  require('../../assets/img/runtime/spring-boot.svg') as string;
-const ThorntailLogo =
-  require('../../assets/img/runtime/thorntail.svg') as string;
-const TomcatLogo = require('../../assets/img/runtime/tomcat.svg') as string;
-const VertxLogo = require('../../assets/img/runtime/vertx.svg') as string;
 
 const renderLogo = (
   name: string,

--- a/plugins/kiali/src/components/MTls/MTLSIcon.tsx
+++ b/plugins/kiali/src/components/MTls/MTLSIcon.tsx
@@ -2,13 +2,10 @@ import * as React from 'react';
 
 import { Tooltip, TooltipPosition } from '@patternfly/react-core';
 
-const fullIcon = require('../../assets/img/mtls-status-full.svg') as string;
-const hollowIcon =
-  require('../../assets/img/mtls-status-partial.svg') as string;
-const fullIconDark =
-  require('../../assets/img/mtls-status-full-dark.svg') as string;
-const hollowIconDark =
-  require('../../assets/img/mtls-status-partial-dark.svg') as string;
+import fullIconDark from '../../assets/img/mtls-status-full-dark.svg';
+import fullIcon from '../../assets/img/mtls-status-full.svg';
+import hollowIconDark from '../../assets/img/mtls-status-partial-dark.svg';
+import hollowIcon from '../../assets/img/mtls-status-partial.svg';
 
 export { fullIcon, hollowIcon, fullIconDark, hollowIconDark };
 

